core/src/test/scala/org/apache/spark/sql/delta/DeltaThrowableSuite.scala

```diff
@@ -53,12 +50,7 @@ class DeltaThrowableSuite extends SparkFunSuite {
   }
 
   test("No error classes are shared by Delta and Spark") {
-    val mapper = JsonMapper.builder().addModule(DefaultScalaModule).build()
-    val sparkErrorClassSource: URL =
-      Utils.getSparkClassLoader.getResource("error/error-classes.json")
-    val sparkErrorClasses = mapper.readValue(
-      sparkErrorClassSource, new TypeReference[Map[String, ErrorInfo]]() {})
-    assert(deltaErrorClassToInfoMap.keySet.intersect(sparkErrorClasses.keySet).isEmpty)
+    assert(deltaErrorClassToInfoMap.keySet.intersect(sparkErrorClassesMap.keySet).isEmpty)
```

Comment:
why this test change? does this test fail without the code change because of the issue we are trying to fix?